### DISCUSSION TITLE
Units of log variables

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#965] [Q&A]
Hi, I want to know what is y direction unit time(kms) ? thanks 
![VirtualBox_BitcrazeVM_13_10_2023_09_45_23](https://github.com/bitcraze/discussions/assets/118436188/aa6f65b1-91c3-4b43-bf79-50005621ef85)

**Comment by knmcguire** [comment]
Ah.. that is indeed not clear at all. This is the logging documentation that comes with it but it doesn't say: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/#ranging. There seems to be a bug in the automated doc generation.

I found the [origin of the log ](https://github.com/bitcraze/crazyflie-firmware/blob/9343aa686600aa5f04beb43549c81bdfda1f9cb8/src/deck/drivers/src/lpsTwrTag.c#L624) and it should be in meters.
